pkg/component/networking/vpn/shoot/shoot.go

```diff
@@ -89,6 +89,12 @@ type Values struct {
 	ReversedVPN ReversedVPNValues
 	// SeedPodNetwork is the pod CIDR of the seed.
 	SeedPodNetwork string
+	// ShootPodNetwork is the pod CIDR of the shoot.
+	ShootPodNetwork string
+	// ShootServiceNetwork is the service CIDR of the shoot.
+	ShootServiceNetwork string
+	// ShootNodeNetwork is the node CIDR of the shoot.
+	ShootNodeNetwork string
```

Comment:
The name suggest something that the values do not cover. Please make the names more explicit.
In a dual-stack scenario, it is not obvious what this will be.